workflows/raredisease.nf

```diff
@@ -32,18 +26,27 @@ def mandatoryParams = [
     "variant_catalog",
     "variant_caller"
 ]
+def missingParamsCount = 0
+
+if (params.run_rtgvcfeval) {
```

Comment:
Perhaps the plugin `nf-validation` could take over some of that "parameter validation" in a future release?